source/developer/slash-commands.rst

```diff
@@ -41,8 +41,8 @@ Each Mattermost installation comes with some built-in slash commands that are re
     "/invite_people *{name@domain.com ...}*", "Send an email invite to your Mattermost team","/invite_people john@example.com"
     "/kick *{@username}*", "Remove a member from a public or private channel", "/kick @alice"
     "/remove *{@username}*", "Remove a member from a public or private channel", "/remove @alice"
-    "/join *{channel-name}*", "Join the open channel", "/join off-topic"
-    "/open *{channel-name}*", "Join the open channel", "/open off-topic"
+    "/join *{channel-name}*", "Join the given channel", "/join off-topic"
```

Comment:
`"/join (or /open) *{channel-name}*", "Join the given channel", "/join off-topic"`